src/components/Tag.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import './tag.sass'
+
+interface TagProps {
+	value?: string
+	size?: string
+}
+
+export default class Tag extends React.PureComponent<TagProps> {
+	render() {
+		if (!this.props.value) {
+			return null
+		}
```

Comment:
I would rather make `value` a required prop. Parent component shouldnt render `<Tag />` for empty values.